[core]/es_extended/server/main.lua

```diff
@@ -377,6 +377,8 @@ AddEventHandler('playerDropped', function(reason)
 
 	if xPlayer then
 		TriggerEvent('esx:playerDropped', playerId, reason)
+        local job = xPlayer.getJob().name
+        ESX.JobsPlayerCount[job] = (ESX.JobsPlayerCount[job] or 1) -1
```

Comment:
Handle please minus values. 
For example:
ESX.JobsPlayerCount["police"] = 0
ESX.JobsPlayerCount["police"] = (ESX.JobsPlayerCount["police"] or 1) -1  

ESX.JobsPlayerCount["police"] -- returns 0 
ESX.JobsPlayerCount["police"] - 1 --  returns 0-1 = -1

Reply:
I'm not sure I understand what you mean.

Reply:
It tracks any users job, so how could someone still have a police job if the count is 0?

Reply:
I have pushed a change anyways.